CONTRIBUTING.md

```diff
@@ -24,6 +59,16 @@ including any errors.
 If you would rather have the KSQL server logs spool to the console, then
 drop the `-daemon` switch, and start the CLI in a second console.
 
+#### Running in an IDE
+
+You can create a run configuration in your IDE for the main class:
+`io.confluent.ksql.rest.server.KsqlServerMain`, using the classpath
+of the `ksqldb-rest-app` module, and specifying the path to a config
+file as the program argument. There is a basic config available in
+`config/ksql-server.properties`.
+
+You may wish to be sure you have already configured your IDE to use
+the Maven wrapper (see "About the Apache Maven wrapper", above).
```

Comment:
Unrelated, but I felt it was a bummer to have to read through `ksql-server-start` to figure out which class to run in IDEA.